services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx

```diff
@@ -13,16 +13,18 @@ You should have received a copy of the MIT License
 along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
 
 Copyright 2023 freiheit.com*/
-import { addAction, useFilteredEnvironmentLockIDs } from '../../utils/store';
+import { addAction, useEnvironmentPriorityClassName, useFilteredEnvironmentLockIDs } from '../../utils/store';
 import { Button } from '../button';
 import { Locks } from '../../../images';
 import * as React from 'react';
 import { EnvironmentLockDisplay } from '../EnvironmentLockDisplay/EnvironmentLockDisplay';
 import { EnvironmentGroup } from '../../../api/api';
+import classNames from 'classnames';
 
 export const EnvironmentCard: React.FC<{ environment: string }> = (props) => {
     const { environment } = props;
     const locks = useFilteredEnvironmentLockIDs(environment);
+    const priorityClassName = useEnvironmentPriorityClassName(environment);
```

Comment:
`useEnvironmentPriorityClassName` should not be a hook.
Suggestion: Make a `useEnvironment` Function
```
const useEnvironment = (string)=>Environment ...
```
Then write another function to get the priorityClassname, e.g.
```
const getPriorityClassName = (env:Environment):string => ...
```